src/components/DeviceDetails/SensorDeleteModal.tsx

```diff
@@ -0,0 +1,155 @@
+import React, { useState } from 'react';
+import { Close } from '@amsterdam/asc-assets';
+import { Modal as DefaultModal, Button, Heading, Paragraph } from '@amsterdam/asc-ui';
+import styled from 'styled-components';
+import { getSpacing } from '../../utils/spacing';
+import CONFIGURATION from '../../shared/environment';
+
+interface SensorDeleteModalProps {
+  open: boolean;
+  onClose: () => void;
+  sensorId: string;
+  email: string;
+}
+
+const SensorDeleteModal: React.FC<SensorDeleteModalProps> = ({ open, onClose, sensorId, email }) => {
+  const [successMessage, setSuccessMessage] = useState<string | null>(null);
+
+  const handleDeleteConfirm = async () => {
+    try {
+      // TODO: Send delete request to the backend with sensorID
+      // const response = await fetch(`${CONFIGURATION.API_ROOT}deleteDevice/`, {
+      //   method: 'DELETE',
+      //   headers: {
+      //     'Content-Type': 'application/json',
+      //   },
+      //   body: JSON.stringify({ sensorId }),
+      // });
+
+      // if (!response.ok) {
+      //   throw new Error('Failed to delete the sensor.'); // Handle error response
+      // }
+
+      console.log('Sensor deleted successfully');
+      setSuccessMessage('Uw aanvraag is verstuurd.');
+    } catch (error) {
+      console.error('Er ging iets mis:', error);
+    } finally {
+      setTimeout(() => {
+        setSuccessMessage(null); // Clear message after a delay
+        onClose();
+      }, 3000);
+    }
+  };
+
+  if (!open) return null;
+
+  return (
+    <StyledModal open={open} onClose={onClose} zIndexOffset={500}>
+      <ModalHeader>
+        <ModalTitle>Bevestig verzoek om sensor te verwijderen</ModalTitle>
+        <CloseButton
+          variant="blank"
+          icon={<Close />}
+          iconSize={16}
+          onClick={onClose}
+        />
+      </ModalHeader>
+
+      <HorizontalRule />
+
+      <ModalContent>
+        <ParagraphStyled>
+          Weet je zeker dat je deze sensor wilt verwijderen? Dit kan niet ongedaan worden gemaakt.
+          We sturen een email naar de eigenaar om de sensor te verwijderen op {email}.
```

Comment:
Is dit mailadres al openbaar voor iedereen te zien? (volgens mij wel he, maar dubbelcheck). Zo niet dan zou ik het hier ook niet noemen.

Reply:
Goed punt. Het staat er inderdaad al volledig in, maar ik had het besproken met een teamlid en we vonden dat die misschien toch niet volledig zichtbaar zou moeten zijn. Dus totdat dat besproken is met de stakeholders dachten we het zo te houden tot we het met de stakeholders besproken hebben.